core/src/test/java/com/yandex/yoctodb/util/buf/FileChannelBufferTest.java

```diff
@@ -19,7 +21,12 @@
 import java.nio.file.Path;
 import java.nio.file.StandardOpenOption;
 import java.util.concurrent.atomic.AtomicInteger;
-import static org.mockito.Mockito.*;
+
+import static org.junit.Assert.assertEquals;
```

Comment:
Someday we should migrate to JUnit 5. Maybe not now...

Reply:
Mm, this is from JUnit 4. I didn't understand the comment properly :)